Automattic/wp-calypso
# Checkout: update 2-year plan upsell card text

Related to 1201210512993648-as-1204738167856704

## Proposed Changes

* Update the text in the plan upsell card for 2-year plans

## Testing Instructions



* Visit the Calypso live link
* Go to /checkout/jetpack/jetpack_security_t1_yearly#step2
* Make sure the card on the right side looks like the one below

<img width="339" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/db9f5a2e-9b81-408d-aa0c-d6e3737a4df0">

## Pre-merge Checklist
